third_party/cri-tools.BUILD

```diff
@@ -0,0 +1 @@
+exports_files(["crictl"])
```

Comment:
Can you think of an other path for this than `third_party`? Somewhere under build?
I'm not sure if it makes sense to wait for approval from third_party folks for deb/rpm changes in general.